nf_core/synced_repo.py

```diff
@@ -260,6 +260,11 @@ def checkout(self, commit):
                 log.debug(f"Overwriting local changes in '{self.local_repo_dir}'")
                 self.repo.git.checkout(self.branch, force=True)
             else:
+                log.error(
+                    f"Could not check out commit '{commit}' in '{self.remote_url}'.\n"
+                    "To solve this, you can try to remove the cloned rempository and run the command again."
+                    f"This repository is tipically found at `~/.config/nfcore/{self.local_repo_dir}`"
```

Comment:
```suggestion
                    f"This repository is found at `{Path(self.local_repo_dir).absolute()}`"
```

Reply:
I'm not sure about this 🤔 Do you know how could we test it? how to create a corrupted repo so that we can see this message

Reply:
yes, just put something in your `~/.config/nfcore/modules` directory

Reply:
ok tested :) it was already an absolute path so we can use it as it is. thanks!